packages/ndla-ui/src/locale/messages-nb.js

```diff
@@ -669,19 +669,22 @@ const messages = {
     text:
       'Har du lyst til å bli god til å presentere, eller vil du lære å studere smartere ved hjelp av riktig studieteknikk? Trenger du råd om hvordan du leser mest mulig effektivt til eksamen? I verktøykassen til NDLA finner du masse gode tips og råd!',
     linkText: 'Se alle tipsene i verktøykassen her',
+    cursorText: 'Tips',
   },
   frontpageMultidisciplinarySubject: {
     heading: 'Tverrfaglig tema',
     text:
       'De tre tverrfaglige temaene i læreplanverket tar utgangspunkt i aktuelle samfunnsutfordringer som krever engasjement og innsats fra enkeltmennesker og fellesskapet i lokalsamfunnet, nasjonalt og globalt.',
-    linkText: 'Få oversikten over våre tverrfaglige tema her',
+    linkText: 'Se caser for tverrfaglige tema',
```

Comment:
Har vi ikkje et bedre ord enn case på norsk? @ghveem?

Reply:
Og skal den endres på bokmål må den også endres på nynorsk.

Reply:
Case står nok i ordboka både på nynorsk og bokmål. Me har ikkje noko betre ord på verken bokmål eller nynorsk inntil vidare. Språkansvarlig har fått beskjed om bruken i alle fall. 

**nn: ein case, casen, fleire casar, alle casane.**
**nb: en case, casen, flere caser, alle casene.**

NN: case m1 (keis; engelsk, av latin casus 'tilfelle') døme (I), eksempel, tilfelle; verkeleg tilhøve.
NB: case m1 (keis; engelsk, fra latin casus 'tilfelle') eksempel, tilfelle, faktiske forhold

Reply:
Greit nok, men då må tilsvarende tekst rettes opp i nn-versjonen.

Reply:
"Sjå casar for tverrfaglege tema" - er den ok?

Reply:
Skal jeg evt endre engelsk og? den er pt "Get an overview of our interdisciplinary topics here"